pkg/templates/templates/schema.json

```diff
@@ -233,6 +233,13 @@
       "description": "Dependent bundles that must be executed with the bundle",
       "type": "object"
     },
+    "required": {
+      "description": "Required extensions for this bundle",
+      "type": "array",
+      "items": {
+        "anyOf": []
```

Comment:
Since they can only use extensions that we know, do we want to list the known extensions here with the config supported? This would give users autocomplete in VS Code.

Reply:
Good idea.  For now, should we list the extensions and config _and also_ allow additionalProperties (so users may define their own).  Or lock it down w/ `additionalProperties: false`?

Reply:
I think that may be a good idea for now, help people with autocomplete but don't lock it down completely. 👍